tizen/src/options.h

```diff
@@ -0,0 +1,59 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_OPTIONS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_OPTIONS_H_
+
+enum class ExistingWorkPolicy { kReplace, kKeep, kAppend, kUpdate };
+
+enum class NetworkType {
+    kConnected,
+    kMetered,
+    kNotRequired,
+    kNotRoaming,
+    kUnmetered,
+    kTemporarilyUnmetered,
+};
+
+enum class OutOfQuotaPolicy {
+    kRunAsNonExpeditedWorkRequest,
+    kDropWorkRequest,
+};
+
+enum class BackoffPolicy { kExponential, kLinear };
+
+struct Constraints {
+    const NetworkType network_type;
+
+    const bool requires_battery_not_low;
+
+    const bool requires_charging;
+
+    const bool requires_device_idle;  // Not supported
+
+    const bool requires_storage_not_low;  // Not supported
+
+    Constraints(NetworkType network_type, bool requires_battery_not_low = false,
+                bool requires_charging = false,
+                bool requires_device_idle = false,
+                bool requires_storage_not_low = false)
+        : network_type(network_type),
+          requires_battery_not_low(requires_battery_not_low),
+          requires_charging(requires_charging),
+          requires_device_idle(requires_device_idle),
+          requires_storage_not_low(requires_storage_not_low) {}
+};
+
+struct TaskType {
+    TaskType(int32_t minimum_backoff_delay)
+        : minimum_backoff_delay(minimum_backoff_delay){};
+    int32_t minimum_backoff_delay;
+    enum { kOneOff = 10000, kPeriodic = 15 * 60 * 1000 };
+};
```

Comment:
The name and function of the `struct` do not match.

Do you need a structure that simply stores `minimum_backoff_delay`?

Reply:
Fixed it!